sites/all/themes/roman_tselepnev/templates/node--even.tpl.php

```diff
@@ -0,0 +1,3 @@
+<div class="even-logo-roman"></div>
+
+<?php include 'node.tpl.php' ?>
```

Comment:
replace include with include_once
https://www.drupal.org/node/318#includes
put semicolon at the end of expression

in general it is not a good practice to use includes within .tpl files, however leave it for now.